components/Heading/Heading.js

```diff
@@ -8,9 +8,8 @@ class Heading extends Component {
   static propTypes = {
     className: string,
     id: oneOfType([string, number]), // reference for scroll anchors
-    anchorId: string,
     customAnchorClass: string,
-    children: node.isRequired,
+    children: string.isRequired,
```

Comment:
You'll want to make this _not_ `children` and call it something else, so people don't accidentally pass JSX and break your dynamic anchorId generation.

Reply:
The props previously contained anchorId and you mentioned that we can remove that by making the children a string. If I were to rename it something else then I would need to add back another prop like anchorId since it's currently just taking the text as the id. Did you want it reverted?

Reply:
children is a reserved prop name for React components. Just because we’ve listed it’s prop type as string doesn’t mean people will adhere to it. In that same message, I suggested renaming your `children` prop to `text` does that make sense?

Then you would be doing `<Heading text=“Who We Serve” headingLevel={1} />`